IGQIUpdater - Extend the Add and Update method (#5040)
SkylineCommunications/dataminer-docs: develop/devguide/GQI Extensions/API_Reference/IGQIUpdater.md

### void AddRow(GQIRow)

> Adds a new row to the query result. If a row with the same row key already exists, all cells will be updated instead. This method effectively acts as an *add or update* mechanism.

Reviewer: Note that it not only updates cells but also metadata, and possibly other row attributes in the future.

```suggestion
Adds a new row to the query result. If a row with the same row key already exists, that row will be updated instead. This method effectively acts as an *add-or-update* operation.
```

### void UpdateRow(GQIRow)

> Updates all cells of an existing row in the query result. If the specified row does not exist yet, the update will be queued and applied once the row is added.

Reviewer: Same here. The previous docs were not quite up-to-date.

```suggestion
Updates an existing row in the query result. If a row with a matching row key does not exist yet, the update event will be queued and applied once the row is added.
```
